config/routes.rb

```diff
@@ -42,6 +42,6 @@
 
   get "/generate_204", to: "application#generate_204", status: 204
 
-  root to: "oauth/applications#index"
+  root to: "data_provider#edit"
```

Comment:
was steckt hier hinter? finde es etwas merkwürdig bei der root route auf eine edit zu kommen ohne passende url dazu.
ich füge einen commit hinzu, wodurch / auf /accounts weiterleitet. oder gäbe es eine bessere idee?

Reply:
52ea7de0d9dbfabf31379fb0c7fd509327267b9d